README.rst

```diff
@@ -17,6 +17,40 @@ For more details `mirrormanager <https://fedorahosted.org/mirrormanager/>`_
 Hacking
 -------
 
+Hacking with Vagrant
+~~~~~~~~~~~~~~~~~~~~
+Quickly start hacking on mirrormanager2 using the vagrant setup that is included
+in the repo is super simple.
+
+First, make a copy of the Vagrantfile example::
+
+    $ cp Vagrantfile.example Vagrantfile
```

Comment:
I think the idea here is to open it up if people do want to change the vagrant file, they can -- depending on if they want different network settings etc.